### PR TITLE
Prometheus updates

### PR DESCRIPTION
**Pull request by max-jacobs**
Applying application tag to prometheus metrics, using composite meter in tests

### CODE REVIEW

File: beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/model/SqsMessageFileTest.java [hunk 1]
```diff
@@ -98,7 +98,6 @@ private void assertKeys(SqsMessageFile sqsMessageFile, Set<String> specificKeys,
       addAll(commonKeys);
     }};
 
-    log.info("\n" + sqsMessageFile.prettyPrintedMessageContents());
```
Comment (nvitucci) [inline beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/model/SqsMessageFileTest.java]: No longer useful?

File: beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperCleanupIntegrationTest.java [hunk 2]
```diff
@@ -268,10 +270,13 @@ void metrics() throws SQLException {
   }
 
   private void assertMetrics() {
-    MeterRegistry meterRegistry = BeekeeperCleanup.meterRegistry();
-    List<Meter> meters = meterRegistry.getMeters();
-    assertThat(meters).extracting("id", Meter.Id.class).extracting("name")
-        .contains("cleanup-job", "s3-paths-deleted", "s3-" + BytesDeletedReporter.METRIC_NAME);
+    Set<MeterRegistry> meterRegistry = ((CompositeMeterRegistry) BeekeeperCleanup.meterRegistry()).getRegistries();
+    assertThat(meterRegistry).containsExactlyInAnyOrder();
```
Comment (nvitucci) [inline beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperCleanupIntegrationTest.java]: Will this line work along with the following line? This one checks that there are no elements.

File: beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/model/SqsMessageFileTest.java (hunk 1 above)
Comment (max-jacobs) [reply beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/model/SqsMessageFileTest.java]: Don't think so

File: beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperCleanupIntegrationTest.java (hunk 2 above)
Comment (max-jacobs) [reply beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperCleanupIntegrationTest.java]: Fixed

**Comment by massdosage** [comment]
> 
> 
> … registry in tests
> 
> Fixes #

?

File: beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/config/PrometheusConfigFactory.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  public String getPrefix() {
+    return prefix;
+  }
+
+  public void setPrefix(String prefix) {
+    this.prefix = prefix;
+  }
+
+  public PrometheusConfig newInstance() {
+    if (prefix == null) {
```
Comment (massdosage) [inline beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/config/PrometheusConfigFactory.java]: NABD but I think you could assign `prefix` to the default on line 29 and then not have this if check here.

**Pull request merged**